js/src/chat/chat.scss

```diff
@@ -1,4 +1,4 @@
-@import "chat-tools.scss";
+@use "chat-tools.scss";
```

Comment:
We're using Dart Sass to compile this file; this avoids a warning because `@import` is going away in a future version.